From: Giovanni Mascellani <gio@debian.org>
Date: Wed, 24 Oct 2018 18:45:25 +0200
Subject: Fix build with Boost 1.67.

boost::asio::io_service requires header boost/asio/io_service.hpp.
---
 src/Connections/CConnectSerialAsio.h | 1 +
 src/Connections/CConnectTCPAsio.h    | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/Connections/CConnectSerialAsio.h b/src/Connections/CConnectSerialAsio.h
index c6a6577..03232f6 100644
--- a/src/Connections/CConnectSerialAsio.h
+++ b/src/Connections/CConnectSerialAsio.h
@@ -43,6 +43,7 @@ Copyright (C) 2010-2012 Tobias Frost
 
 #include <boost/asio/serial_port.hpp>
 #include <boost/asio/streambuf.hpp>
+#include <boost/asio/io_service.hpp>
 #include <semaphore.h>
 
 #include "interfaces/IConnect.h"
diff --git a/src/Connections/CConnectTCPAsio.h b/src/Connections/CConnectTCPAsio.h
index ff5b0ed..afc5d79 100644
--- a/src/Connections/CConnectTCPAsio.h
+++ b/src/Connections/CConnectTCPAsio.h
@@ -38,6 +38,7 @@ Copyright (C) 2009-2012 Tobias Frost
 
 #include <boost/asio/ip/tcp.hpp>
 #include <boost/asio/streambuf.hpp>
+#include <boost/asio/io_service.hpp>
 #include <semaphore.h>
 
 #include "interfaces/IConnect.h"
